lib/paths/index.ts

```diff
@@ -96,6 +103,12 @@ type UnsafePaths<Type, Options extends Required<PathsOptions>> = Type extends Ty
     : never
   : never;
 
+// prettier-ignore
```

Comment:
Is it still required?

Reply:
No, I just thought it read better - the line length seems to be quite long. I'll remove it though.